backend/src/errors/user.ts

```diff
@@ -0,0 +1,7 @@
+import { CustomError } from "./errors";
```

Comment:
We shouldn't create different error files for the different routes. Instead we want to organize our error files by types. See this [link](https://restfulapi.net/http-status-codes/) for the different REST API codes. Essentially there are 4 error codes that we want to handle: 400, 401, 404, and 500. They correspond to validation, auth, service, and internal. Each should have their own file. In this case you want to create a validation error with status 400 to handle if a req.body is incorrect. If there was something wrong with the actual creation of the user then that should be status 500 and stored in the ./errors/internal.ts file.